run.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+#source /reg/g/pcds/engineering_tools/latest-released/scripts/dev_conda ""
+source /reg/g/pcds/engineering_tools/latest-released/scripts/pcds_conda ""
+
+cd "$SCRIPT_DIR" || exit 1
+
+export PYTHONPATH=/cds/group/pcds/pyps/apps/dev/pythonpath:$PYTHONPATH
+export PYTHONPATH=/cds/home/t/tjohnson/trunk/hutch-python/forks/pcdsdevices:$PYTHONPATH
```

Comment:
Is there no deployment area for this in pyps or similar?  Having this run out of your personal space is a scary thing

Reply:
We will make one. I made a ticket to take care of this a while ago, but have been otherwise engaged: https://jira.slac.stanford.edu/browse/ECS-6490

Adam is going to make a small update to the code, so I think that he will be helping make a release or two, helping clean this up.

Reply:
Ah I misread this comment. Yes, this is something I should check on. I forgot about this.